src/View/Admin/Includes/_show_courses_inscription.html.twig

```diff
@@ -10,6 +10,7 @@
                 <th scope="col">Nom d'un parent</th>
                 <th scope="col">Adresse email</th>
                 <th scope="col">Numéro de téléphone</th>
+                <th></th>
```

Comment:
why empty th tag?

Reply:
It's due to the fact that i added buttons to delete and edit at the and of the table, thoose th tags allow to enhance esthetic of the table by adding a black line above buttons

Reply:
why not ^_^